runtime/vm/JFRChunkWriter.hpp

```diff
@@ -200,6 +205,34 @@ class VM_JFRChunkWriter {
 		return _buildResult;
 	}
 
+	void
+	writeIntermediateJFRChunkToFile()
+	{
+		UDATA written = 0;
+		char fileName[sizeof(intermediateChunkFileName) + 16 + sizeof(".jfr")] = {0};
```

Comment:
sizeof a constant string includes the terminator? - I can't remember,

Reply:
Will the sprintf below not zero the end of the string>? The zeroing here seems unnecessary.

Reply:
yes it includes the null terminator